plugins/git/octo.nix

```diff
@@ -0,0 +1,169 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "octo";
+  originalName = "octo.nvim";
+  defaultPackage = pkgs.vimPlugins.octo-nvim;
+
+  maintainers = [ helpers.maintainers.svl ];
+
+  extraPackages = [ pkgs.gh ];
+
+  settingsOptions = {
+    use_local_fs = helpers.defaultNullOpts.mkBool false ''
+      Use local files on right side of reviews.
+    '';
+
+    enable_builtin = helpers.defaultNullOpts.mkBool false ''
+      Shows a list of builtin actions when no action is provided.
+    '';
+
+    default_remote = helpers.defaultNullOpts.mkListOf types.str ''[ "upstream" "origin" ]'' ''
+      Order to try remotes
+    '';
+
+    ssh_aliases = helpers.defaultNullOpts.mkAttrsOf types.str "{ }" ''
```

Comment:
```suggestion
    ssh_aliases = helpers.defaultNullOpts.mkAttrsOf types.str { } ''
```

Reply:
i see that mkAttrsOf uses default, but mkAttributeSet quotes the default. meaning removing quotes around mkAttributeSet make sense, but is it correct in this case also? also note that other uses quote it

Reply:
We've recently added support for passing actual nix expressions into the various `defaultNullOpts` functions, previously that wasn't supported so most uses still pass in pre-formatted strings.